packages/react-components/src/utils.ts

```diff
@@ -63,3 +63,32 @@ export const getIconFromUrl = (url: string): JSX.Element | undefined => {
   });
   return icon?.[1];
 };
+
+export const getListStrWithSuffix = (
+  data: (string | null | undefined)[] = [],
+  suffix: string = '',
+  format?: (str: string) => string,
+): string =>
+  Array.from(new Set(data))
+    .filter((value): value is string => !!value?.trim())
+    .map((item, idx, arr) => {
+      const itemFormatted = format ? format(item) : item;
+      const itemValue = suffix ? `${itemFormatted} ${suffix}` : itemFormatted;
+      const lastIdx = arr.length - 1;
+      return arr.length === 1 || idx !== lastIdx
+        ? `${itemValue}`
+        : `and ${itemValue}`;
+    })
+    .join(', ');
+
+export const capitalizeText = (str: string = ''): string =>
```

Comment:
This should be called TitleCase. 

I am not sure we should be transforming an admin controlled list. Might want to push back on this requirement. I am also wondering if this should just be a text transform: Capitalise